Example/Example/macOS/Code/BankCard/BankCardTextField.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  BankCardTextField.swift
+//  Example
+//
+//  Created by Artem Starosvetskiy on 16/12/2016.
+//  Copyright © 2016 Artem Starosvetskiy. All rights reserved.
+//
+
+import Cocoa
+
+final class BankCardTextField : NSTextField {
+
+    // MARK: - Override NSObject
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        self.wantsLayer = true
+        layer!.borderColor = type(of: self).borderColor.cgColor
```

Comment:
As long as your 'style' variables are common for both UIViewController and NSViewController you can move it so separate structure with static let 'common' variable where all values set by default. 
It helps you to avoid of storing same set of variables in different classes.